src/views/transaction-details/components/chain-row/chain-row.style.ts

```diff
@@ -0,0 +1,12 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const useChainRowStyles = createUseStyles((theme: Theme) => ({
+  hermezChain: {
+    width: theme.spacing(2.5),
```

Comment:
Using theme spacing to define sizes will produce unexpected size changes when one decides to change the spacing.